static/js/components/ShowClassification.jsx

```diff
@@ -80,6 +92,29 @@ const ClassificationRow = ({ classifications }) => {
 
   const classification = classifications[0];
 
+  const upvoterIds = [];
+  const downvoterIds = [];
+  let upvoteValue = 1;
+  let downvoteValue = -1;
+  let upvoteColor = "action";
+  let downvoteColor = "action";
```

Comment:
I would do a green vs red or something here, it would probably make it easier to click the one you want in a hurry.

Reply:
yeah i like that.